Fanny/Fanny.xcodeproj/project.pbxproj

```diff
@@ -658,15 +658,15 @@
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = E5BK5K7RRN;
+				DEVELOPMENT_TEAM = HT8VTXFQ39;
```

Comment:
I need to look into a way to disambiguate this so its not always changing 🤔

Reply:
Maybe I just should just not have committed the xcodeproj? As I did not do any other changes there than the version (I think)

Reply:
That’d work, but I don’t want to have to try to enforce that on every PR.